Mindbox.Expressions/ExpressionExpander.cs

```diff
@@ -166,11 +167,13 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
 				var constantExpression = (ConstantExpression)baseResult.Object;
 				if (IsEvaluateMethod((MethodInfo)constantExpression.Value))
 				{
-					var innerExpression = TryGetLambdaExpressionFromExpression(baseResult.Arguments[1]);
-					if (innerExpression != null)
-						return Visit(ExpressionParameterSubstitutor.SubstituteParameters(
-							innerExpression,
-							new Dictionary<ParameterExpression, Expression>()));
+					var innerExpression = GetLambdaExpressionFromExpression(baseResult.Arguments[1]);
+					if (innerExpression == null)
```

Comment:
Is the null check really needed after every call of `GetLambdaExpressionFromExpression`?

Reply:
It's not :)